### DISCUSSION TITLE
可以给用户请求新插件新增一个issue表单

### DISCUSSION THREAD

**Discussion by QingFeng-awa** [halo-dev/halo#6936] [Ideas]
可以像[LiteLoaderQQNT](https://github.com/LiteLoaderQQNT/LiteLoaderQQNT)(与Halo无关)一样,为请求新插件专门写一个issue表单。
如果认为直接发布到Halo本体不合适也可以考虑在[halo-sigs/awesome-halo](https://github.com/halo-sigs/awesome-halo)里写一个类似表单。

**Comment by JohnNiang** [comment]
Hi @QingFeng-awa ，如果想要提交插件或者主题到 awesome-halo 仓库，可以直接提交 PR 修改列表即可。我们已经提供了对应的 PR 模板，请参考：<https://github.com/halo-sigs/awesome-halo/blob/main/.github/pull_request_template.md>。
